packages/hawtio/package.json

```diff
@@ -24,18 +24,21 @@
   "scripts": {
     "build": "tsup --clean",
     "build:watch": "tsup --watch",
+    "generate:camel-icons": "node ./utils/generate-camel-svg-index.js",
     "test": "jest",
     "test:watch": "jest --watch",
     "test:coverage": "jest --coverage",
     "replace-version": "replace __PACKAGE_VERSION_PLACEHOLDER__ $npm_package_version ./dist/index.js",
     "prepack": "yarn build && yarn replace-version"
   },
   "dependencies": {
+    "@hawtio/camel-model-react": "^0.1.0-dev",
```

Comment:
you can specify the version as `"workspace:*"`. it should be resolved at build time by yarn workspace.